src/appmixer/utils/csv/AddRow/AddRow.js

```diff
@@ -48,13 +49,18 @@ module.exports = {
                     rowAsArray[i] = '';
                 }
             }
-
+            lockExtendInterval = setInterval(async () => {
+                if (lock) {
+                    await lock.extend(parseInt(context.config.lockExtendTime, 10) || 1000 * 60 * 1);
+                }
+            }, 30000);
```

Comment:
the `interval` should be either configurable or somehow related to the `context.config.lockExtendTime`. If you change the `lockExtendTime` to 10s, then the interval will not work correctly. Plus the `TTL` should be configurable, too.